titles/install-rhdh-ocp/master.adoc

```diff
@@ -6,4 +6,28 @@ include::artifacts/attributes.adoc[]
 :imagesdir: images
 
 // ocp deployment
-include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
+//include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
+
+You can install {product} on {ocp-short} by using one of the following installers:
+
+* *The {product} Operator*
```

Comment:
That might be the right use case for a description list https://docs.asciidoctor.org/asciidoc/latest/lists/description/.